### DISCUSSION TITLE
🗺️ Tell us more about your project

### DISCUSSION THREAD

**Discussion by aycabas** [microsoft/hack-together-teams#2] [Show and tell]
We are excited to hear more about your project! 👏

If you are struggling to find a project idea, we've got you! There is a list of [📃 Top Teams Apps Scenarios](https://github.com/microsoft/hack-together-teams/blob/main/TOP-SCENARIOS.md) available! You may build one of these scenarios to win [🏆 Judging and Prizes](https://github.com/microsoft/hack-together-teams#judging-and-prizes-) or get inspired and build your own project ideas!

Don't forget to submit your project until **June 15th 23:59 PM PST**! You can submit your project here when it's ready: [🚀 Project Submission](https://github.com/microsoft/hack-together-teams/issues/new?assignees=&labels=&template=project.yml&title=Project%3A+%3Cshort+description%3E)
Check out this video for step by step project submission guidance: 

https://github.com/microsoft/hack-together-teams/assets/3199282/2310d100-f337-4233-995e-457e6382c182

**Comment by reconsumeralization** [comment]
Title: MeetingsAI - Empowering Productive Meetings with AI-Powered Assistance

Description:
MeetingsAI is a revolutionary AI-powered Teams app designed to transform your meetings into highly productive and efficient collaborative sessions. Harnessing the power of artificial intelligence, MeetingsAI brings advanced features and intelligent assistance to the Microsoft Teams platform, ensuring seamless integration and enhanced user experience.

With MeetingsAI, you can unlock a new level of meeting efficiency and effectiveness. The app combines conversational AI chatbot capabilities, predictive analysis for document and data needs, productivity improvement recommendations, meeting management functionalities, document and data preparation, single sign-on integration, adaptive UI, privacy and security compliance, and user feedback for continuous improvement.

Key Features:

Conversational AI Chatbot: Engage with an AI-powered chatbot that understands and responds to user queries, helping you navigate the Teams environment and providing relevant information.

Predictive Analysis for Document/Data Needs: Leverage advanced machine learning models to analyze meeting/project context and predict the type of data or documents that might be required, utilizing Microsoft Graph data.

Productivity Improvement: Receive intelligent suggestions to enhance team productivity, such as optimizing workflows, recommending less busy meeting times, and providing insights on collaboration patterns.

Meeting Management: Efficiently manage meetings within Teams, including setting agendas, taking notes, determining the sequence of speakers, and performing follow-ups.

Document and Data Preparation: Leverage Power Apps integration to prepare or fetch relevant documents and data based on the context of conversations or meetings.

Single Sign-On (SSO) Integration: Seamlessly access necessary files across various Microsoft services by implementing Single Sign-On (SSO) for all users.

Adaptive UI: Experience an interface that adapts dynamically based on the user's current task or project, presenting the most relevant tools and data.

Privacy and Security: Meeting strict privacy and security guidelines, MeetingsAI ensures data is accessed and used compliantly, safeguarding sensitive information.

User Feedback and Continuous Improvement: Incorporate a system for gathering user feedback to enhance the AI capabilities and overall user experience continuously.

By integrating MeetingsAI into your Microsoft Teams environment, you empower your team with an intelligent meeting assistant that streamlines collaboration, optimizes workflow, and drives productivity. With its intuitive features, advanced AI capabilities, and commitment to privacy and security, MeetingsAI revolutionizes the way teams collaborate, ensuring every meeting is purposeful, efficient, and successful.

Note: This description is a general overview of the idea and can be customized further to highlight specific aspects, benefits, and target audience of the MeetingsAI app.

{
    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.14/MicrosoftTeams.schema.json",
    "manifestVersion": "1.14",
    "version": "1.0.1",
    "id": "f8c5d122-0d54-4947-adab-84fb5b4ec691",
    "packageName": "com.example.meetingapp",
    "developer": {
        "name": "WebDUH LLC",
        "websiteUrl": "https://webduhsign.com",
        "privacyUrl": "https://webduhsign.com/privacy",
        "termsOfUseUrl": "https://webduhsign.com/termsofuse"
    },
    "icons": {
        "color": "resources/color.png",
        "outline": "resources/outline.png"
    },
    "name": {
        "short": "MeetingsAI",
        "full": "MeetingsAI"
    },
    "description": {
        "short": "Helping you run productive meetings",
        "full": "MeetingsAI is an AI-powered Teams app developed by WebDUH LLC. It enhances your meetings with advanced features, including conversational AI chatbot capabilities, predictive analysis, meeting management functionalities, document and data preparation, single sign-on integration, adaptive UI, privacy and security compliance, and user feedback for continuous improvement."
    },
    "accentColor": "#0078D7",
    "bots": [
        {
            "botId": "meetingBot",
            "scopes": [
                "personal",
                "team",
                "groupchat"
            ],
            "supportsFiles": false,
            "isNotificationOnly": false,
            "commandLists": [
                {
                    "scopes": [
                        "personal",
                        "team",
                        "groupchat"
                    ],
                    "commands": [
                        {
                            "title": "Help",
                            "description": "Get help and learn about available commands"
                        },
                        {
                            "title": "Agenda",
                            "description": "Manage meeting agenda and topics"
                        },
                        {
                            "title": "Collaboration",
                            "description": "Facilitate real-time collaboration"
                        },
                        {
                            "title": "Insights",
                            "description": "Retrieve meeting insights and analytics"
                        },
                        {
                            "title": "Record Meeting",
                            "description": "Start recording the meeting"
                        },
                        {
                            "title": "Transcribe Meeting",
                            "description": "Transcribe the recorded meeting"
                        }
                    ]
                },
                {
                    "scopes": [
                        "team",
                        "groupchat"
                    ],
                    "commands": [
                        {
                            "title": "Start Meeting",
                            "description": "Start the meeting and enable meeting features"
                        },
                        {
                            "title": "End Meeting",
                            "description": "End the meeting and generate meeting summary"
                        }
                    ]
                }
            ]
        }
    ],
    "composeExtensions": [],
    "configurableTabs": [
        {
            "configurationUrl": "{{{state.fx-resource-frontend-hosting.endpoint}}}{{{state.fx-resource-frontend-hosting.indexPath}}}/config",
            "canUpdateConfiguration": true,
            "scopes": [
                "personal",
                "team",
                "groupchat"
            ],
            "context": [
                "meetingChatTab",
                "meetingDetailsTab",
                "meetingSidePanel"
            ]
        }
    ],
    "permissions": [
        "messageTeamMembers",
        "identity",
        "user.read",
        "files.read",
        "files.readwrite",
        "channelMessage.Send",
        "teamMember.Read.All",
        "teamMember.ReadWrite.All",
        "channelMessage.Read.All",
        "channelMessage.ReadWrite.All",
        "channelMember.Read.All",
        "groupMember.Read.All",
        "groupMember.ReadWrite.All",
        "offline_access",
        "openid",
        "profile",
        "email",
        "files.read.all",
        "files.readwrite.all",
        "notes.read.all",
        "notes.readwrite.all",
        "people.read.all",
        "tasks.readwrite",
        "directory.read.all",
        "TeamsAppInstallation.ReadWriteForTeam",
        "TeamsTab.ReadWriteForTeam",
        "ChannelMessage.Send",
        "ChannelMessage.Read.All",
        "ChannelMessage.Edit.All",
        "ChannelMessage.Delete.All"
    ],
    "webApplicationInfo": {
        "id": "{{config.ai.applicationId}}",
        "resource": "{{config.ai.resourceUrl}}"
    },
    "validDomains": [],
    "isNotificationOnly": false,
    "staticTabs": [
        {
            "entityId": "home",
            "name": "Home",
            "contentUrl": "{{{state.fx-resource-frontend-hosting.endpoint}}}{{{state.fx-resource-frontend-hosting.indexPath}}}",
            "scopes": [
                "personal",
                "team",
                "groupchat"
            ]
        }
    ],
    "authorization": {
        "permissions": {
            "resourceSpecific": [
                {
                    "name": "LiveShareSession.ReadWrite.Chat",
                    "type": "Delegated"
                },
                {
                    "name": "LiveShareSession.ReadWrite.Group",
                    "type": "Delegated"
                },
                {
                    "name": "MeetingStage.Write.Chat",
                    "type": "Delegated"
                },
                {
                    "name": "ChannelMeetingStage.Write.Group",
                    "type": "Delegated"
                }
            ]
        }
    },
    "webApplicationInfo": {
        "id": "7b5d504c-5ddf-4bdf-a635-cb97070f9399",
        "acceptMappedClaims": null,
        "accessTokenAcceptedVersion": 2,
        "addIns": [],
        "allowPublicClient": true,
        "appId": "f8c5d122-0d54-4947-adab-84fb5b4ec691",
        "appRoles": [
            {
                "allowedMemberTypes": [
                    "User"
                ],
                "description": "MeetingTranscribers can transcribe recorded meetings and access the transcriptions.",
                "displayName": "MeetingTranscriber",
                "id": "e6e4a9de-211c-4ae8-91d2-8b8ea26968d1",
                "isEnabled": true,
                "lang": null,
                "origin": "Application",
                "value": "Meeting.Transcriber"
            },
            {
                "allowedMemberTypes": [
                    "User"
                ],
                "description": "MeetingRecorders have the ability to start and stop meeting recordings.",
                "displayName": "MeetingRecorder",
                "id": "048121ff-db4a-40cc-b422-51202e03ac43",
                "isEnabled": true,
                "lang": null,
                "origin": "Application",
                "value": "Meeting.Recorder"
            },
            {
                "allowedMemberTypes": [
                    "User"
                ],
                "description": "MeetingParticipants can join and participate in meetings,
                                "collaborate, and access meeting resources.",
                "displayName": "MeetingParticipants",
                "id": "e9f45af3-0a8f-4436-b7c9-c0ef888c792d",
                "isEnabled": true,
                "lang": null,
                "origin": "Application",
                "value": "Meeting.Participants"
            },
            {
                "allowedMemberTypes": [
                    "User"
                ],
                "description": "MeetingAdmins have the ability to manage and configure meetings within the app.",
                "displayName": "MeetingAdmins",
                "id": "2562fc2f-c153-4947-aaf0-9b49aee6d497",
                "isEnabled": true,
                "lang": null,
                "origin": "Application",
                "value": "Meeting.Admins"
            }
        ],
        "oauth2AllowUrlPathMatching": false,
        "createdDateTime": "2023-05-11T21:31:21Z",
        "description": null,
        "certification": null,
        "disabledByMicrosoftStatus": null,
        "groupMembershipClaims": null,
        "identifierUris": [],
        "informationalUrls": {
            "termsOfService": "https://www.webduhsign.com/TermsOfService",
            "support": null,
            "privacy": "https://www.webduhsign.com/Privacystatement",
            "marketing": null
        },
        "keyCredentials": [],
        "knownClientApplications": [],
        "logoUrl": "https://aadcdn.msftauthimages.net/dbd5a2dd-nszadoetxyd02ml23iz5yhczkvve1urofgmld8yvgno/appbranding/fwfve3vwlzu9j4dvgk2n5zdztn7awiy8ggcybef-yea/1033/bannerlogo?ts=638194385368161607",
        "logoutUrl": null,
        "name": "MeetingsAI",
        "notes": null,
        "oauth2AllowIdTokenImplicitFlow": false,
        "oauth2AllowImplicitFlow": false,
        "oauth2Permissions": [],
        "oauth2RequirePostResponse": false,
        "optionalClaims": null,
        "orgRestrictions": [],
        "parentalControlSettings": {
            "countriesBlockedForMinors": [],
            "legalAgeGroupRule": "Allow"
        },
        "passwordCredentials": [
            {
                "customKeyIdentifier": null,
                "endDate": "2023-08-09T21:52:00.281Z",
                "keyId": "d0bc4cf5-2a4b-4a55-9b1e-89ae9bf9b192",
                "startDate": "2023-05-11T21:52:00.281Z",
                "value": null,
                "createdOn": "2023-05-11T21:52:20.5859971Z",
                "hint": "34R",
                "displayName": "TeamsHack"
            }
        ],
        "preAuthorizedApplications": [],
        "publisherDomain": "varmsp.onmicrosoft.com",
        "replyUrlsWithType": [
            {
                "url": "msalf8c5d122-0d54-4947-adab-84fb5b4ec691://auth",
                "type": "InstalledClient"
            },
            {
                "url": "https://login.live.com/oauth20_desktop.srf",
                "type": "InstalledClient"
            },
            {
                "url": "https://login.microsoftonline.com/common/oauth2/nativeclient",
                "type": "InstalledClient"
            }
        ],
        "requiredResourceAccess": [
            {
                "resourceAppId": "2d4d3d8e-2be3-4bef-9f87-7875a61c29de",
                "resourceAccess": [
                    {
                        "id": "ab111d9c-370c-4905-a23f-788795f510dc",
                        "type": "Scope"
                    },
                    {
                        "id": "c1c07f53-0865-4ba2-8ca8-36d29aba66f9",
                        "type": "Scope"
                    },
                    {
                        "id": "47eb48ff-1c59-42e3-9634-b5c6e17a0a4b",
                        "type": "Scope"
                    },
                    {
                        "id": "7edff6c9-f308-4b8b-9a6d-847e041e880a",
                        "type": "Scope"
                    },
                    {
                        "id": "93d1f902-06e8-4c6f-8894-1eab0283b521",
                        "type": "Scope"
                    },
                    {
                        "id": "f554dcfe-c273-4d79-9354-4f18f8464b3e",
                        "type": "Scope"
                    }
                ]
            },
            {
                "resourceAppId": "7df0a125-d3be-4c96-aa54-591f83ff541c",
                "resourceAccess": [
                    {
                        "id": "44d8c55c-9ffc-4546-883e-9041b5bb0b01",
                        "type": "Scope"
                    },
                    {
                        "id": "d05743e4-fb24-4dff-8a33-0f6c73c964bd",
                        "type": "Scope"
                    },
                    {
                        "id": "e9cbd5de-0031-493e-8fb1-60712d03cc8b",
                        "type": "Scope"
                    },
                    {
                        "id": "822a9cde-503a-472d-a530-d1dc9cd0d52b",
                        "type": "Scope"
                    },
                    {
                        "id": "30b2d850-00c3-4802-b7ae-ece9af9de5c6",
                        "type": "Scope"
                    },
                    {
                        "id": "e45c5562-459d-4d1b-8148-83eb1b6dcf83",
                        "type": "Scope"
                    },
                    {
                        "id": "0b575251-2c95-46df-bc75-c8c43e6b7116",
                        "type": "Scope"
                    },
                    {
                        "id": "27c61d54-eb42-4315-8793-6e5715a19746",
                        "type": "Scope"
                    }
                ]
            },
            {
                "resourceAppId": "dcad865d-9257-4521-ad4d-bae3e137b345",
                "resourceAccess": [
                    {
                        "id": "47d9c114-7cc8-47c6-9e62-4c5955b05905",
                        "type": "Scope"
                    }
                ]
            },
            {
                "resourceAppId": "c5393580-f805-4401-95e8-94b7a6ef2fc2",
                "resourceAccess": [
                    {
                        "id": "4807a72c-ad38-4250-94c9-4eabfe26cd55",
                        "type": "Scope"
                    },
                    {
                        "id": "e2cea78f-e743-4d8f-a16a-75b629a038ae",
                        "type": "Scope"
                    }
                ]
            },
            {
                "resourceAppId": "499b84ac-1321-427f-aa17-267ca6975798",
                "resourceAccess": [
                    {
                        "id": "ee69721e-6c3a-468f-a9ec-302d16a4c599",
                        "type": "Scope"
                    }
                ]
            },
            {
                "resourceAppId": "1fd5118e-2576-4263-8130-9503064c837a",
                "resourceAccess": [
                    {
                        "id": "6290af7f-b407-49f9-92d5-bf584fdc4019",
                        "type": "Scope"
                    },
                    {
                        "id": "de8ec1df-066a-4817-bc5d-9a985b986262",
                        "type": "Scope"
                    }
                ]
            },
            {
                "resourceAppId": "00000003-0000-0000-c000-000000000000",
                "resourceAccess": [
                    {
                        "id": "e1fe6dd8-ba31-4d61-89e7-88639da4683d",
                        "type": "Scope"
                    }
                ]
            }
        ],
        "samlMetadataUrl": null,
        "signInUrl": "https://www.webduhsign.com",
        "signInAudience": "AzureADandPersonalMicrosoftAccount",
        "tags": [],
        "tokenEncryptionKeyId": null
    }
}

**Reply by waldekmastykarz** [reply]
Whoa, this is a really cool idea! Can't wait to see it in action! 🚀

**Reply by reconsumeralization** [reply]
I would love some help. Honestly setting up my work flow through our the Microsoft platforms is the hardest part for me!

**Reply by waldekmastykarz** [reply]
Awesome that you shared your idea here. Maybe there'll be some folks interested in working together with you. And if you're stuck on anything specific, you can ask a question at https://github.com/microsoft/hack-together-teams/discussions/new?category=q-a at any time and we'll try to get you an answer asap.
